pkg/apis/messaging/v1alpha1/subscription_types.go

```diff
@@ -123,13 +122,7 @@ type ReplyStrategy struct {
 	//  with a reference to the Addressable duck type. If the resource does not meet this contract,
 	//  it will be reflected in the Subscription's status.
 	// +optional
-	*duckv1beta1.Destination `json:",inline"`
-
-	//  The resource pointed by this ObjectReference must meet the Addressable contract
-	//  with a reference to the Addressable duck type. If the resource does not meet this contract,
-	//  it will be reflected in the Subscription's status.
-	// +optional
-	DeprecatedChannel *duckv1beta1.Destination `json:"channel,omitempty"`
+	*duckv1.Destination `json:",inline"`
```

Comment:
We can probably get rid of ReplyStrategy now.

Reply:
yep, done.